src/generated/resources/data/gtceu/tags/items/field_generators.json

```diff
@@ -0,0 +1,17 @@
+{
+  "values": [
+    "gtceu:lv_field_generator",
+    "gtceu:mv_field_generator",
+    "gtceu:hv_field_generator",
+    "gtceu:ev_field_generator",
+    "gtceu:iv_field_generator",
+    "gtceu:luv_field_generator",
+    "gtceu:zpm_field_generator",
+    "gtceu:uv_field_generator",
+    "gtceu:uhv_field_generator",
+    "gtceu:uev_field_generator",
+    "gtceu:uiv_field_generator",
+    "gtceu:uxv_field_generator",
+    "gtceu:opv_field_generator"
```

Comment:
the high-tier ones need to be marked as optional tag entries so the game won't error.